internal/handler/msg_saslstart.go

```diff
@@ -129,3 +149,122 @@ func saslStartPlain(doc *types.Document) (string, string, error) {
 
 	return string(authcid), string(passwd), nil
 }
+
+func (h *Handler) credentialLookup(ctx context.Context, username, dbName string) (scram.StoredCredentials, error) {
```

Comment:
Let's have comment for this function :hugs:

Reply:
Added.